### PR TITLE
Front page

### PR DESCRIPTION
**Pull request by Ardaleon**

### CODE REVIEW

**Review by Mr-Leshiy — COMMENTED** [review]
@Ardaleon general all good and we can merge it, but I have found some interesting behavior with the "downloading" buttons, download for apple is always dark and change they color when it is chosen, download for android button vice versa, is it intended ? Maybe we should have a similar behavior for them ?
@SultanOfWing What do you think ?

**Review by Mr-Leshiy — COMMENTED** [review]
Remove Tezos image

File: src/components/UI/HeadSocials/HeadSocials.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <ul className={styles.socials}>
+      <li>
+        <a href="https://github.com/">
+          <img src={icon_linkedin} alt="linkedin" />
+        </a>
+      </li>
+      <li>
+        <a href="https://github.com/">
+          <img src={icon_inst} alt="linkedin" />
+        </a>
```
Comment (SultanOfWing) [inline src/components/UI/HeadSocials/HeadSocials.jsx:18]: why linkedin?

File: src/components/UI/HeadSocials/HeadSocials.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <ul className={styles.socials}>
+      <li>
+        <a href="https://github.com/">
+          <img src={icon_linkedin} alt="linkedin" />
+        </a>
+      </li>
+      <li>
+        <a href="https://github.com/">
+          <img src={icon_inst} alt="linkedin" />
```
Comment (SultanOfWing) [inline src/components/UI/HeadSocials/HeadSocials.jsx]: why linkedin? should be inst

File: src/components/UI/HeadSocials/HeadSocials.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        </a>
+      </li>
+      <li>
+        <a href="https://github.com/">
+          <img src={icon_telegram} alt="linkedin" />
```
Comment (SultanOfWing) [inline src/components/UI/HeadSocials/HeadSocials.jsx]: why linkedin? should be telegram

File: src/components/elements/About/About.jsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+import image_2 from "../../../assets/images/image-about-2.jpg";
+import image_3 from "../../../assets/images/image-about-3.jpg";
+import image_4 from "../../../assets/images/image-about-4.jpg";
+
+const About = () => {
+  return (
+    <section className={styles.about}>
+      <Title classes="title_brown">About Us</Title>
+
+      <div className={styles.about__container}>
```
Comment (Mr-Leshiy) [inline src/components/elements/About/About.jsx:21]: Combine all divs into one with one className with the combined styles

File: src/components/elements/About/About.jsx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+        <div className={styles.about__item}>
+          <div className={styles.about__top}>
+            <div className={styles.about__image_wrapper}>
+              <img src={image_1} alt="tickets" />
+            </div>
+          </div>
+          <div className={styles.about__bottom}>
```
Comment (Mr-Leshiy) [inline src/components/elements/About/About.jsx:28]: Same

File: src/components/elements/About/About.jsx [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+                <p>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                  do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div className={styles.about__item_mobile_reversed}>
```
Comment (Mr-Leshiy) [inline src/components/elements/About/About.jsx:46]: Same

File: src/components/elements/About/About.jsx [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+            </div>
+          </div>
+          <div className={styles.about__bottom}>
+            <div className={styles.about__image_wrapper}>
+              <img src={image_2} alt="QR" />
+            </div>
+          </div>
+        </div>
+
+        <div className={styles.about__item}>
```
Comment (Mr-Leshiy) [inline src/components/elements/About/About.jsx:70]: Same

File: src/components/elements/About/About.jsx [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+          <div className={styles.about__top}>
+            <div className={styles.about__image_wrapper}>
+              <img src={image_3} alt="NFT" />
+            </div>
+          </div>
+          <div className={styles.about__bottom}>
```
Comment (Mr-Leshiy) [inline src/components/elements/About/About.jsx:76]: Same

File: src/components/elements/About/About.jsx [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+                <p>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                  do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div className={styles.about__item_mobile_reversed}>
```
Comment (Mr-Leshiy) [inline src/components/elements/About/About.jsx:94]: Same

File: src/components/elements/About/About.jsx [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+                  </h3>
+                </div>
+                <p>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                  do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div className={styles.about__bottom}>
```
Comment (Mr-Leshiy) [inline src/components/elements/About/About.jsx:111]: Same

File: src/components/elements/Footer/Footer.jsx [hunk 11]
```diff
@@ -0,0 +1,52 @@
+import Button from "../../UI/Button/Button";
+import FooterSocials from "../../UI/FooterSocials/FooterSocials";
+import React from "react";
+import styles from "./Footer.module.scss";
+
+const Footer = () => {
+  return (
+    <footer className={styles.footer}>
+      <div className={styles.footer__main}>
```
Comment (Mr-Leshiy) [inline src/components/elements/Footer/Footer.jsx:9]: Same

File: src/components/elements/Footer/Footer.jsx [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+            <li>
+              <a href="https://github.com/">Info</a>
+            </li>
+          </ul>
+
+          <FooterSocials></FooterSocials>
+        </div>
+      </div>
+
+      <div className={styles.footer__bottom}>
```
Comment (Mr-Leshiy) [inline src/components/elements/Footer/Footer.jsx:29]: Same

File: src/components/elements/Slider/Slider.jsx [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+
+const Slider = () => {
+  return (
+    <section className={styles.slider__section}>
+      <Title classes="title_brown">How to install?</Title>
+
+      <div className={styles.slider__container}>
+        {dataSlider.map((obj, index) => {
+          return (
+            <div className={styles.slider__slide}>
```
Comment (Mr-Leshiy) [inline src/components/elements/Slider/Slider.jsx:14]: Same

File: src/components/elements/Footer/Footer.jsx [hunk 14]
```diff
@@ -0,0 +1,52 @@
...
+        <div className={styles.footer__bottom_wrapper}>
+          <span className={styles.footer__copywrite}>
+            ©2022 Все права защищены
```
Comment (SultanOfWing) [inline src/components/elements/Footer/Footer.jsx]: use english plz

File: src/components/elements/Slider/Slider.jsx [hunk 15]
```diff
@@ -0,0 +1,46 @@
...
+              <div className={styles.slider__slide_image}>
+                <img
+                  alt=""
+                  src={
+                    process.env.PUBLIC_URL +
+                    `/slider/slider-image-${index + 1}.svg`
```
Comment (SultanOfWing) [inline src/components/elements/Slider/Slider.jsx:20]: could it be a constant?

File: src/components/elements/About/About.jsx (hunk 4 above)
Comment (Ardaleon) [reply src/components/elements/About/About.jsx:21]: I need clarification and one code example.

**Comment by Ardaleon** [comment]
> @Ardaleon general all good and we can merge it, but I have found some interesting behavior with the "downloading" buttons, download for apple is always dark and change they color when it is chosen, download for android button vice versa, is it intended ? Maybe we should have a similar behavior for them ? @SultanOfWing What do you think ?

Depends on what kind of effect you want these buttons have. Given they're different as it is i guess we can only turn hover off for black button.

File: src/components/elements/Slider/Slider.jsx (hunk 15 above)
Comment (Ardaleon) [reply src/components/elements/Slider/Slider.jsx:20]: "process.env.PUBLIC_URL" this one?

File: src/components/elements/About/About.jsx (hunk 4 above)
Comment (Mr-Leshiy) [reply src/components/elements/About/About.jsx:21]: Something like that 
```
 <div className={`${styles.card} ${props.className}`}>{props.children}</div>;
 ```

**Review by Mr-Leshiy — APPROVED** [review]

**Pull request merged**